event_track_assistant/models/event.py

```diff
@@ -57,6 +57,97 @@ def onchange_date_end(self):
                 self.date_end = track.date
         return res
 
+    @api.multi
+    def button_mass_mailing_to_organizer(self):
+        ir_model_data = self.env['ir.model.data']
+        if not self.organizer_id.email:
+            raise exceptions.Warning(
+                _("Organizer %s without email.") % self.organizer_id.name)
+        if self.organizer_id.notify_email != 'always':
+            self.organizer_id.notify_email = 'always'
+        try:
+            template_id = self.env.ref(
+                'event_track_assistant.email_to_event_organizer', False)
+        except ValueError:
+            template_id = False
+        try:
+            compose_form_id = ir_model_data.get_object_reference(
```

Comment:
Y aquí lo mismo, no haría lo siguiente...
`compose_form_id = ir_model_data.get_object_reference('mail', 'email_compose_message_wizard_form') and ir_model_data.get_object_reference('mail', 'email_compose_message_wizard_form')[1] or False`